.gitignore

```diff
@@ -57,3 +57,5 @@ lib/*/*.lib
 lib/*/lib/*.lib
 
 lib/qtscript-bytearray/*.cc
+
+*.vscode
```

Comment:
This file is created by Mixxx or its build-system. Adding patterns for every conceivable editor/IDE to this project's gitignore is neither desirable nor feasible. You should add it to your [global gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore) instead.

Reply:
This was already mentioned in that PR ages ago. I can't seem to find it.